src/Cake/Commands/HelpCommand.cs

```diff
@@ -28,6 +30,8 @@ public bool Execute(CakeOptions options)
             _console.WriteLine();
             _console.WriteLine("Options:");
             _console.WriteLine("    -verbosity=value    Specifies the amount of information to be displayed.");
+            _console.WriteLine("                        Verbosities: {0}",
+                string.Join(", ", Enum.GetNames(typeof(Verbosity))));
```

Comment:
I would prefer to add the verbosities directly here instead of enumerting them. Makes it easier to see how the formatting will look like as well.

```
-verbosity=value    Specifies the amount of information to be displayed.
                    (diagnostic, verbose, normal, minimal, quiet)
```